wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -439,6 +442,35 @@ public void clone(CloneRequest request) throws GitException, UnauthorizedExcepti
                 cloneCommand.setBranchesToClone(request.getBranchesToFetch());
             }
 
+            LineConsumer lineConsumer = lineConsumerFactory.newLineConsumer();
+            cloneCommand.setProgressMonitor(new BatchingProgressMonitor() {
+                @Override
+                protected void onUpdate(String taskName, int workCurr) {
+                    try {
+                        lineConsumer.writeLine(taskName + ": " + workCurr + " completed");
```

Comment:
We have one TCK test suite for both Git implementations. I think it is not good to test there this progress monitor because it is a part of implementation and it doesn't perform any git operations.